crates/cxx-qt-lib-extras/build.rs

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use cxx_qt_build::CxxQtBuilder;
+use qt_build_utils::QtBuild;
+
+fn main() {
+    let mut builder = CxxQtBuilder::new();
+
+    // Find the Qt version and tell the Rust compiler
+    // this allows us to have conditional Rust code
+    //
+    // TODO: is this useful to have in cxx-qt-build?
```

Comment:
@LeonMatthesKDAB any thoughts on this ? I wonder if in `cxx-qt-build` we should always set the `qt_version_major` as a `rustc-cfg` option so that we don't need to do it in `cxx-qt-lib` and `cxx-qt-lib-extras` ? And then even apps could use it? Or do we want to just confine this option to the `libs` for now so that it is not a "supported API" ?

Reply:
Indeed sounds useful to have in `cxx-qt-build`!
We should always know the Qt major version, so shouldn't be a problem to support this.

The only thing I'd do is prefix all cfg options with `cxxqt`.
So: `cxxqt_qt_version_major`, such that we don't collide with cfg options from other crates.

Not `cxxqt_version_major`, as that could be confused with the version of CXX-Qt itself.

Reply:
So this has landed, so you should be able to remove these lines. Note that it's now not called `qt_version_major` though.